rpm/qemu.spec

```diff
@@ -1547,10 +1545,40 @@ QEMU.
 %_datadir/%name/vhost-user/50-qemu-gpu.json
 %_libexecdir/vhost-user-gpu
 
+%package img
+Summary:        QEMU disk image utility
+Group:          System/Emulators/PC
+
+%description img
+This package provides a command line tool for manipulating disk images.
```

Comment:
Maybe reword to "provides command line tools" since the package contains more than one tool.

Reply:
Right, initially I included only `qemu-img` in the package. Then forgot to reflect the update in the description.